<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginsOptions</class>
 <widget class="QWidget" name="PluginsOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugins</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="rawGroup">
     <property name="title">
      <string>Raw scaner</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="rawMinSizeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Minimal data size to scan, bytes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="rawMinSizeValue">
          <property name="minimum">
           <number>128</number>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="rawPlainDoubleAnalysis">
        <property name="toolTip">
         <string>&lt;b&gt;Warning!&lt;/b&gt; This may cause big amount of duplicates while searching in raw data</string>
        </property>
        <property name="text">
         <string>Double analysis of plain data containers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="playersGroup">
     <property name="toolTip">
      <string>Duration for modules without timing info</string>
     </property>
     <property name="title">
      <string>Players</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="defaultDurationLabel">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Default duration, seconds</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="defaultDurationValue">
          <property name="minimum">
           <number>30</number>
          </property>
          <property name="maximum">
           <number>600</number>
          </property>
          <property name="value">
           <number>120</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
